bigquery/bigquerystorage/src/main/java/com/example/bigquerystorage/StorageArrowSample.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquerystorage;
+
+// [START bigquerystorage_quickstart]
```

Comment:
Yes, it should be soonish for filtering, there are still going to be some caveats so I think separate samples makes sense in the short term since at least in java Arrow can be a little harder to work with.

Reply:
updated to include filtering in example.

Reply:
> I think separate samples makes sense in the short term since at least in java Arrow can be a little harder to work with.

In that case, could you update the Avro sample with a new region tag, maybe `bigquerystorage_quickstart_avro`?

Reply:
if they are two separate files is there a strong reason for having different region tags (it would force updating documentation as well as this change)?

Reply:
We treat region tags as globally unique in our sample tracker. Since these are two different samples, it makes sense to track separately. That way we can know where else (which other languages) that are missing one format or the other.

Reply:
Alternatively, use `bigquerystorage_quickstart_arrow` in this file, if we don't want Arrow to be the default yet.